packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -346,7 +341,9 @@ const Step4 = ({ show_label = false }) => (
             </div>
         </div>
         <div>
-            <img src={getImageLocation('bot-builder-tour-step-4.gif')} alt='step4' />
+            <video autoPlay loop playsInline preload='auto' controls style={{ width: '-webkit-fill-available' }}>
```

Comment:
Can we add `width: '-webkit-fill-available'` property in the css level, it has been reused multiple times.

Reply:
@prince-deriv Sure can. We actually don't need this. Will be using width: 100%. I'll add it in the css. Thanks.